lib/got-client.ts

```diff
@@ -0,0 +1,9 @@
+import got, { Options } from 'got';
+
+const options = {} as Options;
+
+if (process.env.NODE_ENV === 'test') {
+  options.retry = 0;
+}
```

Comment:
By default `got` will retry failed requests a couple times. In test env when we're using `nock` we don't really want that to happen